kitsune/questions/tests/test_forms.py

```diff
@@ -56,7 +54,7 @@ def test_metadata_keys(self):
         expected = ['troubleshooting', 'ff_version', 'os',
                     'plugins', 'useragent']
         actual = form.metadata_field_keys
-        eq_(expected, actual)
+        eq_(sorted(expected), sorted(actual))
```

Comment:
Why are you using sorted here?

Reply:
This is what was causing the test to fail - the items were equal, but their order was different. I assume we don't care in which order this metadata appears?